public/script.js

```diff
@@ -1,15 +1,34 @@
 function init() {
-    test();
+    initRef();
 }
 
-function test() {
-    const divObject = document.getElementById('content');
+const path = '/messages'; // can make this more detailed (for example add user ID)
 
+// initializes the .on() functions for the database reference
+function initRef() {
     // create database reference
-    const dbRefObject = firebase.database().ref().child('object');
+    const dbRefObject = firebase.database().ref(path);
 
     // sync object data
+    const divObject = document.getElementById('content');
     dbRefObject.on('value', snap => {
         divObject.innerHTML = JSON.stringify(snap.val(), null, 3);
     });
+
+    const listObject = document.getElementById('chat-as-list');
+    dbRefObject.on('child_added', snap => {
+        const li = document.createElement('li');
+        li.innerText = snap.val();
+        console.log(snap.val());
+        listObject.appendChild(li);
+    });
+}
+
+function pushChatMessage() {
+    const messageRef = firebase.database().ref(path);
+
+    const message = document.getElementById('message-input').value;
+    // push message to datastore
+    messageRef.push(message);
+    console.log("yo")
```

Comment:
Is it possible to avoid merging debug code into master?  Ideally, by the time you make a pull request, the code should be debugged sufficiently that you don't need this any more.

Reply:
you're right, i need to get better about that but I keep forgetting to take it out 🤕